sklearn/feature_selection/univariate_selection.py

```diff
@@ -361,6 +362,9 @@ def fit(self, X, y):
     def _check_params(self, X, y):
         pass
 
+    def _more_tags(self):
+        return {'allow_nan': True}
```

Comment:
@amueller thinks we should change this to False, at least for the default score_func. Then this seems close to merge?